sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -492,6 +574,13 @@ public partial class DistanceScoringParameters
         public double BoostingDistance { get { throw null; } set { } }
         public string ReferencePointParameter { get { throw null; } set { } }
     }
+    public partial class DocumentExtractionSkill : Azure.Search.Documents.Indexes.Models.SearchIndexerSkill
+    {
+        public DocumentExtractionSkill(System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.OutputFieldMappingEntry> outputs) { }
+        public System.Collections.Generic.IDictionary<string, object> Configuration { get { throw null; } }
+        public string DataToExtract { get { throw null; } set { } }
+        public string ParsingMode { get { throw null; } set { } }
```

Comment:
I think this should be an enum which means we probably have a swagger bug?  Fine for this preview, but we should dig into it.